File: java/section3/PersonExample.java
// import section3.ClassPerson;
public class PersonExample {
    public static void main(String[] args) {
        // create object here
        var person1 = new ClassPerson();
        person1.name = "suryo atmojo";
        person1.address = "sukolilo";
        // person1.country secara default mengikuti atribute class

        Person person2 = new ClassPerson();  
        Person person3;
        person3 = new ClassPerson();

        System.out.println(person1.name);
        System.out.println(person1.address);
        System.out.println(person1.country);
        
    }
}

class ClassPerson {
   // Class representation for a person
    // This class is used to store and manipulate person-related information
    String name;
    String address;
    final String country = "indonesia";
}

File: php/section1/PersonExample.php
<?php
class Person {
    public function __toString() {
        return "This is a person object\n";
    }
}
class PersonExample {
    public static function main() {
        // create object here
        $person1 = new Person();
        $person2 = new Person();

        $person3 = new Person();

        echo $person1 . "\n";
        echo $person2 . "\n";
        echo $person3 . "\n";
    }
}

PersonExample::main();
?>
